mycli/main.py

```diff
@@ -85,12 +94,10 @@ def __init__(self, sqlexecute=None, prompt=None,
         if defaults_file:
             self.cnf_files = [defaults_file]
 
-        default_config = os.path.join(PACKAGE_ROOT, 'myclirc')
-        write_default_config(default_config, '~/.myclirc')
-
-
         # Load config.
-        c = self.config = load_config('~/.myclirc', default_config)
+        c = self.config = read_config_file(self.default_config_file)
+        read_config_files(self.system_config_files, base_config=c)
+        read_config_file(self.user_config_file, base_config=c)
```

Comment:
Yeah, I'll fix this. I originally had it set up to not mutate on each call. I changed, but I forgot why.